src/Project.cpp

```diff
@@ -3,28 +3,32 @@
 #include "LDtkLoader/Project.hpp"
 
 #include <fstream>
+#include <sstream>
 
 #include "LDtkLoader/World.hpp"
 #include "json.hpp"
 
 using namespace ldtk;
 
-void Project::loadFromFile(const std::string& filepath) {
+void ldtk::Project::loadFromFile(const std::string& filepath) {
+    loadFromFile(filepath, defaultFileLoader);
```

Comment:
Suggestion from another comment

```suggestion
    auto ifstream_loader = [&] (const std::string& filepath) {
        // copy defaultFileLoader body here
    }
    loadFromFile(filepath, ifstream_loader);
```

This way, we don't need to define a separated method and we keep it near where it is used.